File: calendar_app/lib/models/event.dart
class Event {
  String id;
  List<Event> events;
  String title;
  String date;
  String startTime;
  String endTime;

  Event(this.title, this.date, this.startTime, this.endTime);

  factory Event.fromJson(Map<String, dynamic> parsedJson) {
    return Event(parsedJson['title'], parsedJson['date'],
        parsedJson['start_time'], parsedJson['end_time']);
  }
}

File: TimeTableBackend/api/models/event.js
const mongoose = require('mongoose');

const eventSchema = mongoose.Schema({
    _id: mongoose.Schema.Types.ObjectId,
    title : {type: String , required : true},
    date: { type: String , required : true },
    start_time : { type: String , required : true },
    end_time: { type: String , required : true }
});

module.exports = mongoose.model('EventsList' , eventSchema);